data-es6.js

```diff
@@ -66,12 +66,10 @@ exports.browsers = {
     short: 'IE 11',
     obsolete: false
   },
-  ie11tp: {
-    full: 'Internet Explorer',
-    short: 'IE<br>Technical<br>Preview',
-    unstable: true,
-    note_id: 'ie-experimental-flag',
-    note_html: 'Have to be enabled via "Experimental Web Platform Features" flag'
+  spartan: {
+    full: 'Project Spartan',
```

Comment:
The `full` name for the Trident/Chakra line of browsers has to be changed to "Internet Explorer / Project Spartan", because the "differential results" code in build.js uses the `full` name as a key to determine succession - for instance, setting `ie10: true` causes IE 11 to have a true result as well, because its `full` name is the same and it appears after it in the `browsers` object (which is of course considered ordered by definition order for the purposes of `Object.keys`).

Ideally there'd be a different value, maybe called `family`, that determined this, but currently this is how it is.